.github/workflows/test.yml

```
@@ -4,7 +4,7 @@ on:
   push: # Run on pushes to the default branch
     branches: [main]
   pull_request_target: # Also run on pull requests originated from forks
-    branches: [main]
+    branches: [main, rename-test-conn]
```

Before this PR is merged, we should revert this change:
```suggestion
    branches: [main]
```